packages/mocha-runner/src/MochaOptionsLoader.ts

```diff
@@ -5,41 +5,61 @@ import { tokens, commonTokens } from '@stryker-mutator/api/plugin';
 import { Logger } from '@stryker-mutator/api/logging';
 import { serializeArguments, filterConfig, mochaOptionsKey } from './utils';
 import LibWrapper from './LibWrapper';
+import { MochaOptions } from './MochaOptions';
 
-export default class MochaOptionsLoader {
+/**
+ * Subset of defaults for mocha options
+ * @see https://github.com/mochajs/mocha/blob/master/lib/mocharc.json
+ */
+export const DEFAULT_MOCHA_OPTIONS = Object.freeze({
+  extension: ['js'],
+  opts: './test/mocha.opts',
+  spec: ['test'],
+  timeout: 2000,
+  ui: 'bdd'
+});
 
-  private readonly DEFAULT_MOCHA_OPTS = 'test/mocha.opts';
+export default class MochaOptionsLoader {
 
   public static inject = tokens(commonTokens.logger);
   constructor(private readonly log: Logger) { }
 
   public load(strykerOptions: StrykerOptions): MochaOptions {
     const mochaOptions = { ...strykerOptions[mochaOptionsKey] } as MochaOptions;
-    return { ... this.loadMochaOptions(mochaOptions), ...mochaOptions };
+    return { ...DEFAULT_MOCHA_OPTIONS, ... this.loadMochaOptions(mochaOptions), ...mochaOptions };
   }
 
   private loadMochaOptions(overrides: MochaOptions) {
     if (LibWrapper.loadOptions) {
-      this.log.debug('Mocha > 6 detected. Using mocha\'s `%s` to load mocha options', LibWrapper.loadOptions.name);
-      const args = serializeArguments(overrides);
-      const rawConfig = LibWrapper.loadOptions(args) || {};
-      if (this.log.isTraceEnabled()) {
-        this.log.trace(`Mocha: ${LibWrapper.loadOptions.name}([${args.map(arg => `'${arg}'`).join(',')}]) => ${JSON.stringify(rawConfig)}`);
-      }
-      return filterConfig(rawConfig);
+      this.log.debug('Mocha >= 6 detected. Using mocha\'s `%s` to load mocha options', LibWrapper.loadOptions.name);
+      return this.loadMocha6Options(overrides);
     } else {
       this.log.debug('Mocha < 6 detected. Using custom logic to parse mocha options');
```

Comment:
Didn't we want to log this as a deprecation warning? Or is that somewhere else?

Reply:
Let's do that in a separate PR, before we release 2.0

Reply:
Sure 👍